
#### Software and versions are listed below

- HiFiAdapterFilt [https://github.com/sheinasim/HiFiAdapterFilt](url)
- NanoPlot [https://github.com/wdecoster/NanoPlot](url)
- minimap2/2.21-r1071 [https://github.com/lh3/minimap2](url)
- samtools/1.9 [https://github.com/samtools/samtools](url)
- fastp/0.19.6 [https://github.com/OpenGene/fastp](url)

  #### Process
  
1. PacBio Sequel II ccs.bam file (17G) was downloaded 4 March 2023 and filtered of adaptors using HiFiAdapterFilt. Data available at NCBI accession: 
2. Ran NanoPlot on filtered data to check statistics.
   
General summary:
- Mean read length:                11,370.9
- Mean read quality:                   40.3
- Median read length:              11,103.0
- Median read quality:                 39.1
- Number of reads:              2,214,940.0
- Read length N50:                 11,975.0
- Total bases:             25,185,858,588.0


3. Ramiacotti Centre for Genomics, UNSW, HiC paired-end fastq.gz data (using Phase Genomics library preparation kit) (47G) was downloaded on the 27 April 2023. Raw data available at NCBI accession: .
4. We mapped the .ccs data to the diploid _Austropuccinia psidii_ genome ([https://zenodo.org/records/6476632](url)), and included the mitochondrial sequence, with minimap2 to separate plant from pathogen reads. We then processed .sam to .bam with samtools.

   
   ```
   minimap2 -d ${prefix_name}.mmi ${prefix_name}.fasta
   minimap2 -K 2g -t 8 -ax map-hifi -o $OUTDIR/${prefix_name}.sam -a ${prefix_name}.mmi ccs.filt.fastq.gz
   ```
   
5. Output from the mapping was unmapped.bam (6.8G) and mapped.bam (93M). As the pathogen haploid genome is 1G, the read coverage was too low to continue with genome assembly.
6. We proceeded with the unmapped, filtered ccs reads (_Syzygium luehmannii_) combined with the HiC reads to build the diploid genome.
