
#### Software and versions

- PAFScaff v0.6.2 [https://github.com/slimsuite/pafscaff](url) (run with minimap2/2.21-r1071, python/2.7.9)
- FCS [https://github.com/ncbi/fcs](url)
- taxolotl [https://github.com/slimsuite/taxolotl](url)
- telociraptor v0.8.0 [https://github.com/slimsuite/telociraptor](url)
- chromsyn v1.2.1 [https://github.com/slimsuite/chromsyn](url) (run with outputs from BUSCO v 5.1.2 with lineage dataset embryophyta_odb10 (Creation date: 2024-01-08), R/4.3.2, python/3.6.5)
- hisat2/2.1.0 [https://daehwankimlab.github.io/hisat2/](url)
- samtools/1.9 [https://github.com/samtools/samtools](url)
- seqkit/2.2.0 [https://bioinf.shenwei.me/seqkit/](url)
- busco/5.1.2 [https://busco.ezlab.org/busco_userguide.html](url) (run with lineage dataset embryophyta_odb10 (Creation date: 2024-01-08), hmmsearch/3.2 and metaeuk/a0f584d0f2ee96ab8fd7e6e6367761051d5ef4a5) 

#### Process

1. We ran reference-based scaffold anchoring and super-scaffolding on each of our HiC haploid scaffolded genomes independently with pafscaff, using the NCBI _Melaleuca quinquenervia_ reference genome <https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_030064385.1/>

```
minimap2 --cs -p 0.0001 -t 8 -x asm10 -N 100 $WDIR/${ref}  $WDIR/${assembly} > $WDIR/${prefix}.paf`

python $HOMEDIR/pafscaff/code/pafscaff.py seqin=$WDIR/${assembly} reference=$WDIR/${ref} pafin=$WDIR/${prefix}.paf basefile=$WDIR/${prefix} refprefix=${refprefix} newprefix=${newprefix} unplaced=${unplaced} forks=8
```

2. We concatenated the scaffolded and unplaced fasta files for a complete set of contigs for each haplotyype.

3. Next we ran telociraptor followed by chromsyn on just the chromosome level contigs to compare the synteny between Hap A and B.  Initially the synteny plots indicated some errors in scaffolding.
 ![Alt text](https://github.com/peritob/Syzygium-luehmannii/blob/main/SL.png)

4. Then the Hap A and B contigmaps (SL.ctgmap.txt) were manually adjusted to conform and telomeres were moved that were marked as { or } to 5' and 3'. Rerun telociraptor with adjust map in and synteny graphs rechecked with chromsyn.

```
python $HOMEDIR/telociraptor/code/telociraptor.py seqin=SL_B.chr.fa basefile=SL_B i=-l tweak=T autofix=T mapin=SL_B.ctgmap_adjust.txt telomeres=T
```

After tweaking the synteny was improved.
![Alt text](https://github.com/peritob/Syzygium-luehmannii/blob/main/SL.tweak.png)


9. We then ran the genomes through contamination screening using FCS-GX, taxolotl and tiara and removed contigs that were flagged as non-plant or plastid to retain only genomic DNA sequences

10. We attempted to build complete plastid genomes first using the assembled and flagged organelle contigs and then using all the filtered.ccs.fastq reads with mitohifi/3.0.0. This did not assemble a complete circularised mito or chloroplast genome. We speculate that this might be due to the HMW DNA extraction protocol that depletes cytosolic material as well as subsequent size selection for PacBio sequencing. 
    
11. We checked the conserved single copy gene complement of each haploid genome with BUSCO.

```
C:98.2%[S:97.5%,D:0.7%],F:0.8%,M:1.0%,n:1614 - T_cacao_A_v1.3.fasta
C:98.4%[S:97.8%,D:0.6%],F:0.9%,M:0.7%,n:1614 - T_cacao_B_v1.3.fasta
```
    
